docs/collections/dynamo/dynamo-table-settings.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Dynamo Table Settings
+sidebar_position: 18
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ViewCollectionSettings from '../_partials/_view-collection-settings.md';
+import CollectionSettingsApi from '../_partials/_collection-settings-api.md';
+
+<Tabs groupId="operating-systems">
+<TabItem value="ui" label="UI">
+
+You can view and update key-value store collection settings in the Settings tab.
+
+<ViewCollectionSettings />
+
+## Dynamo Table Settings
+
+Dynamo Table settings are displayed as a table schema, similar to the following code block.
```

Comment:
```suggestion
Dynamo Table settings are displayed as a table schema similar to the following code block.
```

Same as other comma suggestion, not necessary but I'm also fine with leaving it if we want it to be more readable to ESL folks.